example/counterBasedRng/src/counterBasedRng.cpp

```diff
@@ -162,8 +162,7 @@ auto main() -> int
     CounterBasedRngKernel::Key key = {rd(), rd()};
 
     // Allocate buffer on the accelerator
-    using BufAcc = alpaka::Buf<Acc, Data, Dim, Idx>;
-    BufAcc bufAcc(alpaka::allocBuf<Data, Idx>(devAcc, extent));
+    auto bufAcc(alpaka::allocBuf<Data, Idx>(devAcc, extent));
```

Comment:
Personally, I prefer the syntax
```suggestion
    auto bufAcc = alpaka::allocBuf<Data, Idx>(devAcc, extent);
```

It's just a preference, not a request to make any changes.